README.md

```diff
@@ -48,100 +48,96 @@ Your website/app must present information in ways people can recognise and use,
 
 * [Resize text:](guidelines/1.4.4.md) Make sure it is possible to complete all tasks when text is resized up to 200%.
 
-* [Reflow:](guidelines/1.4.10.md) All information and functionality must be available on a screen that's 320 CSS pixel wide, without needing to scroll horizontally
+* [Reflow:](guidelines/1.4.10.md) Make sure users can access all information and functionality on a screen that's as wide as on the iPhone5, without needing to scroll in both directions.
```

Comment:
Nice. I like that this makes it more agnostic and just specifies a device width, which can easily be related to Android and desktop